static/css/devhub/new-landing/base.less

```diff
@@ -31,7 +31,7 @@
 @color-scheme-dark-items: #40d38d;
 
 // Buttons
-@color-button-default: #0568ba;
+@color-button-default: #0996F8;
```

Comment:
This seems to be the intended color from my Sketch files unless I'm mistaken somehow.